test/spec/features/modeling/behavior/DetachEventBehaviorSpec.js

```diff
@@ -61,6 +62,29 @@ describe('features/modeling/behavior - detach events', function() {
       expect(boundaryEvent.type).to.equal('bpmn:BoundaryEvent');
       expect(boundaryEvent.businessObject.attachedToRef).to.equal(subProcess.businessObject);
     }));
+
+
+    it('should execute on pasting a BoundaryEvent outside', inject(function(canvas, elementRegistry, copyPaste) {
```

Comment:
This behavior is executed on `elements.create` (`modeling.createElements`), we don't want to test copy & paste here.

Also, please restructure the tests for readability:

```javascript
describe('basics', function() {
  describe('create elements', function() {
    it('should detach');
    it('should NOT detach');
  });
  describe('move elements', function() {
    it('should detach');
    it('should NOT detach');
  });
});
```